api/http/api_inventory.go

```diff
@@ -42,8 +44,9 @@ const (
 	uriGroups        = "/api/0.1.0/groups"
 	uriGroupsDevices = "/api/0.1.0/groups/:name/devices"
 
-	uriInternalTenants = "/api/internal/v1/inventory/tenants"
-	uriInternalDevices = "/api/internal/v1/inventory/devices"
+	uriInternalTenants       = "/api/internal/v1/inventory/tenants"
+	uriInternalDevices       = "/api/internal/v1/inventory/devices"
+	urlInternalDevicesStatus = "/api/internal/v2/inventory/devices/:tenant_id/:status"
```

Comment:
@merlin-northern to be totally honest, I don't like this URL because it is not semantic, reading the URL:
`/api/internal/v2/inventory/devices/1/status`

I read it as "update the status of device 1". I also don't get why we need to increment the version (v2), as we are not changing the old behaviour, just adding a new end-point. Also, how does this work if you don't have a tenant (e.g., in the open source version)? Does this work?

If the API is "per-tenant", I believe `/api/internal/v1/inventory/tenants/:tenant_id/devices/:status` is a better candidate.

Reply:
yes it works without tenant normally.  I checked last week. I have no problem with changing the url and staying with v1 it was incremented because it is the new way: we accept a set of device ids.

Reply:
I added a [test](https://github.com/mendersoftware/inventory/pull/162/commits/a71f3299d8788c8abb31838f631a53b0ac321f7b#diff-e00c71bbd3855c0ab6891e3a9d7a0dbbR1567) scenario for that.

Reply:
my [pleasure](https://github.com/mendersoftware/inventory/pull/162/commits/a71f3299d8788c8abb31838f631a53b0ac321f7b#diff-d38bb1bb13c34ab0a761f45bb70cd491R49)